packages/cli/src/commands/models/m365Sideloading.ts

```diff
@@ -19,14 +20,22 @@ export const m365SideloadingCommand: CLICommand = {
       name: "file-path",
       description: "Path to the App manifest zip package.",
       type: "string",
-      required: true,
+    },
+    {
+      name: "xml-path",
```

Comment:
Does this only apply to cli v3?
For v2 there's other code at *cmds/m365/m365.ts*.

Reply:
That's a good point. I'll talk with huajie about it.

Reply:
According to @jayzhang , CLI v3 has not been enabled by default.
However, since the command is only used by office-addin tooling, I think it's OK to ask them to enable it when using xml sideload.